packages/clerk-js/src/ui/components/OrganizationSwitcher/OrganizationSwitcherTrigger.tsx

```diff
@@ -33,6 +33,9 @@ export const OrganizationSwitcherTrigger = withAvatarShimmer(
         colorScheme='neutral'
         sx={[t => ({ minHeight: 0, padding: `0 ${t.space.$2} 0 0`, position: 'relative' }), sx]}
         ref={ref}
+        aria-label='organization-switcher-trigger'
```

Comment:
This needs to be something more descriptive like this:

```suggestion
        aria-label={`${props.isOpen ? 'Close' : 'Open'} organization switcher`}
```

Reply:
@LekoArts I get why we want to remove  the `-`, but why do we need to add Close and Open ? Shouldn't we just rely on `aria-expanded` ?

Reply:
I can't link to W3C right now (it's down for me), so for ref the mozilla docs: https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Attributes/aria-expanded#buttons
It's not necessary but a nice-to-have

I think in combination with the haspopup (https://www.accessibility-developer-guide.com/examples/sensible-aria-usage/expanded/#adding-haspopup-optional) this will be better